src/Uno.UWP/Generated/3.0.0.0/Windows.UI.Composition/CompositionTarget.cs

```diff
@@ -8,25 +8,6 @@ namespace Windows.UI.Composition
 #endif
 	public partial class CompositionTarget : global::Windows.UI.Composition.CompositionObject
 	{
-#if __ANDROID__ || __IOS__ || IS_UNIT_TESTS || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
```

Comment:
Undo this change and adjust the one in `Microsoft.UI.Xaml.Media` instead. Then the implementation will need to be inside Uno.UI project, so the `Visual` in `Uno.UI.Composition` will just use plain `object` to reference the instance (as composition project does not have reference to `Uno.UI`)